src/goto-symex/goto_symex.h

```diff
@@ -332,6 +332,9 @@ class goto_symext
   /// Symbolically execute a GOTO instruction
   /// \param state: Symbolic execution state for current instruction
   virtual void symex_goto(statet &state);
+  /// Symbolically execute a GOTO instruction in the context of unreachable code
+  /// \param state: Symbolic execution state for current instruction
+  void symex_unreachable_goto(statet &state);
```

Comment:
I suspect this method can be marked `const` or even `static`

Reply:
Not done: calls `symex_transition` which is itself not const... and so on. Can do constification of the `goto_symext` class but not here.